### PR TITLE
added mbc3 tester

### PR DESCRIPTION
**Pull request by vojty**

### CODE REVIEW

**Review by c-sp — CHANGES_REQUESTED** [review]
Thanks for your efforts and the quite thorough PR!

I would like to suggest the following changes:

- Most test suites' screenshots are created using the color palette described [here](https://github.com/c-sp/gameboy-test-roms/blob/master/src/howto/dmg-acid2.md#test-successfailure). Could you change color `#C0C0C0` to `#AAAAAA`?
- Could you add a screenshot for "CGB compatibility" mode? Replacing color `#C0C0C0` with `#7BFF31` should be sufficient for this.
- Could you rename `game.gb` to e.g. `mbc3-tester.gb`?

**Comment by vojty** [comment]
Sure no problem!

File: src/howto/mbc3-tester.md [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+comparison.
+
+To create comparable screenshots,
+your emulator should calculate colors as follows:
+
+- `#000000`, `#555555`, `#AAAAAA` and `#FFFFFF` are used for the four DMG LCD
+  shades
+- LCD shades for CGB compatibility mode are:
+  - background: `#000000`, `#0063C6`, `#7BFF31` and `#FFFFFF`
+  - objects: `#000000`, `#943939`, `#FF8484` and `#FFFFFF`
```
Comment (vojty) [inline src/howto/mbc3-tester.md:27]: I copy & pasted this line from acid, maybe I can drop the colors for objects

**Comment by vojty** [comment]
fixed

**Comment by c-sp** [comment]
Great, thanks a lot!

No need to adjust the color paragraph in my opinion.

**Comment by vojty** [comment]
alright, let's merge it so I can add the others as well

**Pull request merged**